playbooks/boilerplate.yml

```diff
@@ -7,49 +7,59 @@
 # - to reduce inventory boilerplate (defining parent groups / empty groups)
 
 - name: Add kube-master nodes to kube_control_plane
-  hosts: kube-master
+  hosts: all
   gather_facts: false
   tags: always
   tasks:
     - name: Add nodes to kube_control_plane group
-      group_by:
-        key: 'kube_control_plane'
+      add_host:
+        name: "{{ item }}"
+        groups: 'kube_control_plane'
+      with_items: "{{ groups['kube-master'] | default([]) }}"
 
 - name: Add kube-node nodes to kube_node
-  hosts: kube-node
+  hosts: all
   gather_facts: false
   tags: always
   tasks:
     - name: Add nodes to kube_node group
-      group_by:
-        key: 'kube_node'
+      add_host:
+        name: "{{ item }}"
+        groups: 'kube_node'
+      with_items: "{{ groups['kube-node'] | default([]) }}"
```

Comment:
This loops over the whole cluster, which adds up on big clusters.
You could probably do something like this instead
```yaml
key: "{{ 'kube-node' in group_names | ternary('','not_') }}kube_node"
```

In fact this would probably let us loop over the whole groups and use only one task.